tests/Doctrine/Tests/DBAL/DriverManagerTest.php

```diff
@@ -38,11 +38,11 @@ public function testInvalidPdoInstance()
      */
     public function testValidPdoInstance()
     {
-        $conn = DriverManager::getConnection([
-            'pdo' => new PDO('sqlite::memory:'),
-        ]);
+        $pdo  = new PDO('sqlite::memory:');
```

Comment:
Perhaps we need more tests covering the basic DBAL features for when an instance is used?